src/Dfe.Spi.EntitySquasher/Dfe.Spi.EntitySquasher.AcdfGen.ConsoleApp/Registry.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Dfe.Spi.EntitySquasher.AcdfGen
+{
+    using System.Diagnostics.CodeAnalysis;
+    using Dfe.Spi.Common.Logging.Definitions;
+    using Dfe.Spi.EntitySquasher.AcdfGen.Application;
+    using StructureMap.Graph;
+
+    /// <summary>
+    /// Custom, host-specific implementation of
+    /// <see cref="StructureMap.Registry" />.
+    /// </summary>
+    [ExcludeFromCodeCoverage]
+    public class Registry : StructureMap.Registry
```

Comment:
Would it not make more sense to use MS DI to be consistent

Reply:
Replaced StructureMap with the default MS/.NET Core one.